pkgs/development/php-packages/smbclient/default.nix

```diff
@@ -0,0 +1,13 @@
+{ buildPecl, lib, pkgs }:
```

Comment:
```suggestion
{ buildPecl, lib, samba, pkg-config }:
```

Reply:
@etu @talyz @Ma27 did we want to switch all the `php` packages and extensions over to this style?

Reply:
Well, we could, but maybe not worth to change all packages and extensions in this PR. It's more commonly used across nixpkgs so it would be nice to follow that.

Reply:
Great. I'll write up a PR once I get a chance. Thanks!

Reply:
Overwriting packages is not that easy when they are used indirectly.